charts/enterprise-logs/tools/update-version

```diff
@@ -0,0 +1,19 @@
+#!/usr/bin/env bash
+# Update references to Grafana Enterprise Metrics versions
```

Comment:
I actually forgot this script existed in the enterprise-metrics chart...
It looks like this should work fine here too, only this comment is incorrect.